subprojects/core/src/main/java/org/gradle/api/internal/changedetection/state/InputFileSnapshotNormalizationStrategy.java

```diff
@@ -23,7 +23,7 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-public enum TaskFilePropertySnapshotNormalizationStrategy implements SnapshotNormalizationStrategy {
+public enum InputFileSnapshotNormalizationStrategy implements SnapshotNormalizationStrategy {
```

Comment:
I did quite some renames. I saw that you opened an issue for renaming `PathSensitivity` to `PathNormalization`, so I will not do anything about this in the current PR.